src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpOverallOutcomeController.cs

```diff
@@ -0,0 +1,239 @@
+                case ApplicationStatus.InProgress:
+                    return RedirectToAction("TaskList", "RoatpApplication", new {applicationId});
+                case ApplicationStatus.Approved:
+                {
+                    var oversightReview = await _apiClient.GetOversightReview(applicationId);
+                    if (oversightReview?.Status == OversightReviewStatus.SuccessfulAlreadyActive)
+                        return RedirectToAction("ApplicationApprovedAlreadyActive", new {applicationId});
+
+                    return RedirectToAction("ApplicationApproved", new {applicationId});
+                }
+                case ApplicationStatus.Rejected: //this logic will need to change with the coming status update story
+                    if (application.GatewayReviewStatus == GatewayReviewStatus.Fail)
+                        return RedirectToAction("ApplicationUnsuccessful", new {applicationId});
+                    return RedirectToAction("ApplicationUnsuccessful",  new {applicationId});
+                case ApplicationStatus.FeedbackAdded:
+                    return RedirectToAction("FeedbackAdded", new {applicationId});
+                case ApplicationStatus.Withdrawn:
+                    return RedirectToAction("ApplicationWithdrawn", new {applicationId});
+                case ApplicationStatus.Removed:
+                    return RedirectToAction("ApplicationRemoved",  new {applicationId});
+                case ApplicationStatus.GatewayAssessed:
+                    if (application.GatewayReviewStatus == GatewayReviewStatus.Reject)
+                        return RedirectToAction("ApplicationRejected", 
+                            new {applicationId});
+                    return RedirectToAction("ApplicationSubmitted",  new {applicationId});
+                case ApplicationStatus.Submitted:
+                case ApplicationStatus.Resubmitted:
+                    return RedirectToAction("ApplicationSubmitted",  new {applicationId});
+                default:
+                    return RedirectToAction("TaskList", "RoatpApplication", new {applicationId});
+            }
+        }
+
+        [HttpGet]
+        [Route("ApplicationStatus")]
+        [Authorize(Policy = "AccessApplication")]
+        public async Task<IActionResult> ApplicationUnsuccessful(Guid applicationId)
+        {
+            var application = await _apiClient.GetApplication(applicationId);
+            var applicationData = application.ApplyData.ApplyDetails;
+
+            var oversightReview = await _apiClient.GetOversightReview(applicationId);
+            
+
+            // this will change based on coming stories and overturns etc
+            var applicationGatewayReviewStatusFail = (application.GatewayReviewStatus == GatewayReviewStatus.Fail);
+
+            if (applicationGatewayReviewStatusFail)
+            {
+                var unsuccessfulModel = await BuildApplicationSummaryViewModel(applicationId);
```

Comment:
Something to pick up later but going via this is calling GetApplication and we have it above. It might be with building the Summary View Model prior to this. This may go in hand with the comment further down.